cmake/CMakeLists.txt

```diff
@@ -3,3 +3,53 @@ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/lib/opencog.conf
 # Create a test configuration file by copying the default configuration file
 CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/lib/opencog-test.conf
     ${PROJECT_BINARY_DIR}/lib/opencog-test.conf)
+
+# install Find* cmake files to the DATADIR
+INSTALL(FILES
+        AddCxxtest.cmake        
+        FindBFD.cmake
+        FindCxxtest.cmake
+        FindCython.cmake
+        FindGTK3.cmake
+        FindGuile.cmake
+        FindHyperTable.cmake
+        FindIberty.cmake
+        FindLibGearman.cmake
+        FindMsgPack.cmake
+        FindPthreads.cmake
+        FindStack.cmake
+        FindSTLPort.cmake
+        FindVALGRIND.cmake
+        FindZMQ.cmake
+        FindSIGAR.cmake
+        FindPython3Interp.cmake
+        FindPGSQL.cmake
+        FindTBB.cmake
+        FindBFD.cmake
+        Summary.cmake
+        OpenCogFunctions.cmake
+        OpenCogAtomTypes.cmake
+        OpenCogMacros.cmake
+        DESTINATION
+        ${DATADIR}/cmake/)
+
+set(INCLUDE_INSTALL_DIR include/ )
+set(LIB_INSTALL_DIR lib/ )
+
+include(CMakePackageConfigHelpers)
+configure_package_config_file(CogUtilConfig.cmake.in ${CMAKE_CURRENT_BINARY_DIR}/CogUtilConfig.cmake
+                              INSTALL_DESTINATION ${LIB_INSTALL_DIR}/CogUtil/cmake
+                              PATH_VARS INCLUDE_INSTALL_DIR LIB_INSTALL_DIR)
+
+write_basic_package_version_file(
+  "${CMAKE_CURRENT_BINARY_DIR}/CogUtilConfigVersion.cmake"
+  VERSION ${SEMANTIC_VERSION}
+  COMPATIBILITY SameMajorVersion
+)
+
+INSTALL(FILES
+       ${CMAKE_CURRENT_BINARY_DIR}/CogUtilConfigVersion.cmake
+       ${CMAKE_CURRENT_BINARY_DIR}/CogUtilConfig.cmake
+       DESTINATION
+       ${CMAKE_INSTALL_PREFIX})
```

Comment:
Should be ${LIB_INSTALL_DIR}/CogUtil/cmake probably

Reply:
Thanks!